packages/editor/src/components/post-text-editor/style.scss

```diff
@@ -1,27 +1,28 @@
 .editor-post-text-editor {
-	border: $border-width solid $light-gray-500;
+	border: $border-width solid $light-gray-500 !important;
 	display: block;
 	margin: 0 0 2em;
 	width: 100%;
 	box-shadow: none;
 	resize: none;
 	overflow: hidden;
-	font-family: $editor-html-font;
+	font-family: $editor-html-font !important;
 	line-height: 150%;
+	border-radius: 0 !important;
 
 	/* Fonts smaller than 16px causes mobile safari to zoom. */
-	font-size: $mobile-text-min-font-size;
+	font-size: $mobile-text-min-font-size !important;
 	@include break-small {
-		font-size: $text-editor-font-size;
+		font-size: $text-editor-font-size !important;
 	}
 
 	&:hover,
 	&:focus {
-		border: $border-width solid $light-gray-500;
-		box-shadow: none;
+		border: $border-width solid $light-gray-500 !important;
+		box-shadow: none !important;
 		// Emulate the effect used on the post title.
-		outline: $border-width solid $light-gray-500;
-		outline-offset: -2px;
+		outline: $border-width solid $light-gray-500 !important;
```

Comment:
This outline results in a `2px` overall border now, which seems weird to me: 

![hover](https://user-images.githubusercontent.com/1202812/55401505-fc319b00-551e-11e9-98cf-8ba6b7537eff.gif)

I think we can probably remove this outline, and make the hover/focus border color `$light-gray-800` to match the selected block border color.